distribution/bwc/build.gradle

```diff
@@ -106,9 +110,14 @@ if (enabled) {
 
   // this is an Exec task so that the SHA that is checked out is logged
   task checkoutBwcBranch(type: Exec) {
-    dependsOn fetchLatest
+    def String branch = System.getProperty("tests.bwc.branch", "upstream/${bwcBranch}")
+    if (branch.startsWith('upstream')) {
+      dependsOn fetchLatest
```

Comment:
That's a good idea  - it simplifies things and make them more powerful. I'll do that. Thanks.